tests/routes/test_ingress.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+from connect.clients import kafka, nats
+from connect.config import get_settings
+from connect.workflows.core import CoreWorkflow
+from unittest.mock import AsyncMock
+
+
+@pytest.mark.parametrize(
+    "fixture_name,data_format",
+    [
+        ("x12_fixture", "X12-005010"),
+        ("fhir_fixture", "FHIR-R4"),
+        ("hl7_fixture", "HL7-V2"),
+    ],
+)
+@pytest.mark.asyncio
+async def test_ingress_post_x12(
+    fixture_name,
```

Comment:
Minor point, but should the test now be called "test_ingress_post" because it's not x12-specific?

Reply:
@ccorley - yes thanks for the catch! I will update the name before merging.